com.vbam.visualboyadvancem.yaml

```diff
@@ -0,0 +1,119 @@
+app-id: com.vbam.visualboyadvancem
```

Comment:
`vbam.com` doesn't look related to this. Is this a domain controlled by the upstream developers?

Reply:
Sadly it's vba-m.com but when I tried to build it, flatpak-builder was complaining about the dash for some reason.

As for ffmpeg. I opted for a more minimalistic build over the full bodied compile the SDK provides.

Reply:
because `-` is to be replaced with `_`. One doesn't simply take a domain from someone else.

there is no point in building ffmpeg. It's just a waste of space.

Reply:
specifically vba-m.com being the domain, but flatpak-builder doesn't like that. 

``error: 'com.vba-m.visualboyadvance-m' is not a valid application name: Only last name segment can contain -``
``error: 'com.vba-m.visualboyadvancem' is not a valid application name: Only last name segment can contain -``

Reply:
Also I'm one of the upstream developers.

Reply:
as I just said, replace `-` with `_`.

https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
> there is no point in building ffmpeg. It's just a waste of space.

Fine, I get it. Also, apologies. didn't see the previous comment. I'll resolve those issues in a moment.

Reply:
Alright, new commit should resolve most of these current issues.